RecoTracker/TkTrackingRegions/plugins/PointSeededTrackingRegionsProducer.h

```diff
@@ -62,10 +64,11 @@ class  PointSeededTrackingRegionsProducer : public TrackingRegionProducer
     else if (modeString == "VerticesSigma") m_mode = VERTICES_SIGMA;
     else  edm::LogError ("PointSeededTrackingRegionsProducer")<<"Unknown mode string: "<<modeString;
 
-    // basic inputs
-    edm::ParameterSet point_input = regPSet.getParameter<edm::ParameterSet>("point_input");
-    eta_input          = point_input.getParameter<double>("eta");
-    phi_input          = point_input.getParameter<double>("phi");
+    // basic inputsi
+    edm::ParameterSet points = regPSet.getParameter<edm::ParameterSet>("points");
+    etaPoints = points.getParameter<std::vector<double>>("eta");
+    phiPoints = points.getParameter<std::vector<double>>("phi");
```

Comment:
I see your point; I agree a vector of PSet makes more sense, but indeed it should be converted into a vector of doubles, or better into a vector of "coordinates", defined for example as 
```
struct EtaPhi {
  double eta;
  double phi;
}
```